### PR TITLE
HomeTask70 v1.0

### PR DESCRIPTION
**Pull request by MaksimKuznets**
Develop 2 tests:

1. Login to yandex.com (username – seleniumtesttraining@yandex.com; password – 1qaz@WSX1qaz, create your own record if needed)

2. Logout from yandex.com

### CODE REVIEW

File: homeTask70/HomeTask70/BasePage.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public abstract class BasePage
+    {
+        protected IWebDriver Driver;
+
+        protected BasePage(IWebDriver driver)
+        {
+            this.Driver = driver;
+        }
+
+        protected IWebElement FindElenemtByAnyMethod(By selector)
```
Comment (alexsavenok-savenduk) [inline homeTask70/HomeTask70/BasePage.cs]: FindElement*

File: homeTask70/HomeTask70/BasePage.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            IWebElement element = null;
+            try
+            {
+                Waiting.EplicitWaiting(Driver, 5, selector);
+                 element = this.Driver.FindElement(selector);
+
```
Comment (alexsavenok-savenduk) [inline homeTask70/HomeTask70/BasePage.cs]: Remove empty lines within your methods

File: homeTask70/HomeTask70/BasePage.cs (hunk 1 above)
Comment (MaksimKuznets) [reply homeTask70/HomeTask70/BasePage.cs]: Done

File: homeTask70/HomeTask70/BasePage.cs (hunk 2 above)
Comment (MaksimKuznets) [reply homeTask70/HomeTask70/BasePage.cs]: Done

**Pull request merged**